wsproto/connection.py

```diff
@@ -393,6 +393,34 @@ def _process_connection_request(self, event):
 
         return ConnectionRequested(proposed_subprotocols, event)
 
+    def extension_accept(self, extensions_header):
```

Comment:
Are you suggesting it should be renamed to _extension_accept or that it should go back into the function I pulled it out of?

I'm fine with the former.